### PR TITLE
Fix: clarify that security team is made of the community

### PR DESCRIPTION
**Pull request by xmulligan**
Signed-off-by: Bill Mulligan <billmulligan516@gmail.com>

### CODE REVIEW

**Comment by xmulligan** [comment]
TODO: link to roles page on how to get involved

**Review by tgraf — APPROVED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -16,5 +16,5 @@ We strongly encourage you to report security vulnerabilities to
 our private security mailing list: security@cilium.io - first, before
 disclosing them in any public forums.
 
-This is a private mailing list where only members of the Cilium internal
-security team are subscribed to, and is treated as top priority.
+This is a private mailing list where community members from the project
+across multiple organization are subscribed to, and is treated as top priority.
```
Comment (lizrice) [inline SECURITY.md:20]:
```suggestion
This is a private mailing list where community members from the project security team (from 
across multiple organizations) are subscribed, and is treated as top priority.
```

**Review by lizrice — COMMENTED** [review]
suggestion that we clarify it is a security team, not a random group of community members. (We need to define the security team in the contributor ladder as well but that's a separate issue)

**Comment by xmulligan** [comment]
closing in favor of https://github.com/cilium/cilium/pull/24135

**Pull request closed without merging**